docroot/sites/archaeology/settings.php

```diff
@@ -790,8 +790,13 @@
 #   include $app_root . '/' . $site_path . '/settings.local.php';
 # }
 
-if (file_exists('/var/www/site-php')) {
-  require '/var/www/site-php/swshumsci/archaeology-settings.inc';
```

Comment:
This is all handled in BLT's black magic now? I don't see this require anywhere else in this pr?

Reply:
yep. more obscurity https://github.com/acquia/blt/blob/10.x/settings/blt.settings.php#L125